### DISCUSSION TITLE
Why don't you create a telegram group ?!

### DISCUSSION THREAD

**Discussion by ghost** [hecrj/iced#1939] [Ideas]
Iced team and users are using the closed source (malware) Discord for fast communication and help, why don't you create a telegram group beside using discord, telegram desktop client is much faster (written in C++) and more resource efficient than discord, plus it's open source and not a malware.

**Comment by ple1n** [comment]
Telegram is malware. 

Use https://matrix.org/

**Reply by ghost** [reply]
The client side of telegram is not malware cuz the apps are open source, however the server code is closed source, so they are probably doing some shit with the user's data such as selling it. Tho you can exploit their fast server speed to your advantage and only use end to end encryption (secret chats and calls), anything you wanna make absolutely private use E2EE. Other than that content that you don't care if they sell like a public iced group is not a problem.

Matrix is both open source in both client and server, and that makes it amazing. 2 problems it has is that the server speed and storage space is not as good as telegram, plus the element app is written in the shitty slow resource eating javascipt. Telegram desktop app is written in C++ and is amazingly fast.

**Comment by Redhawk18** [comment]
you can create your own unofficial areas

**Reply by ghost** [reply]
I am zero knowledge in iced, so it won't be appropriate for me to create the group.
